### ISSUE TITLE
Comma-separated values not working (except font-family)

### ISSUE THREAD

**Issue by erickreutz** [less/less.ruby#53]
background: url(image) no-repeat top, url(image) no-repeat bottom; gives a error at compile.. It is part of the css3 spec

**Comment by mcvsama** [comment]
This is also true for text-shadow. Less doesn't like commas in property values?

**Comment by zuf** [comment]
another example of this bug:

```
 -moz-box-shadow: 0pt 0pt 2px rgba(255, 255, 255, 0.4) inset,
 0pt 4px 6px rgba(255, 255, 255, 0.4) inset;
```

**Comment by cloudhead** [comment]
fixed in 1.2.1 (github only)
